Applications/yOCTScanAndFindTissueSurface.m

```diff
@@ -1,44 +1,148 @@
-function [surfacePosition_mm, x_mm, y_mm] = yOCTScanAndFindTissueSurface(varargin)
+function [surfacePosition_mm, x_mm, y_mm, isSurfaceInFocus] = yOCTScanAndFindTissueSurface(varargin)
 % This function uses the OCT to scan and then identify tissue surface from 
 % the OCT image.
-% INPUTS (all inputs are parameters):
 %   xRange_mm, yRange_mm - what range to scan, default [-1 1] mm.
-%   resolution_mm - pixel resolution, default: 0.050 mm.
+%   pixel_size_um - Pixel resolution for this analysis, default: 15 um.
```

Comment:
I think in our call we said 25 microns. Am I wrong?

Reply:
Yes! Sorry, I was trying to make sure the merge wouldn't cause problems, but I saw you did it, thanks! I updated pixel sizes now.

Since it's 25 as default, should we simply remove it from the calling function on Demo_ScanAndProcess_3D?